icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -303,6 +308,8 @@ void dumpScriptExtensions(FILE* f) {
     const char* scxShortPropName = u_getPropertyName(UCHAR_SCRIPT_EXTENSIONS, U_SHORT_PROPERTY_NAME);
     fprintf(f, "long_name = \"%s\"\n", scxFullPropName);
     if (scxShortPropName) fprintf(f, "short_name = \"%s\"\n", scxShortPropName);
+    // enum props are all hex values after 0x1000
```

Comment:
Nit (optional): wrong comment?

Alternatively, just use hex for everything, including binary properties, and then you don't need to justify why you use hex for these

Reply:
Done.